data/human/human missions.txt

```diff
@@ -7455,3 +7455,1860 @@ mission "Adelita 5"
+			`	As your ship approaches the mine you can't help but notice the burnt out remains of a gutted and abandoned town. A few minutes later your ship rattles down outside the mine's entrance.`
+			choice
+				`	(Leave your ship.)`
+			`	It's dark and dusty outside. The planet's gravity makes you feel slow, and the air uncomfortably heavy in your lungs. The entrance to the mine looms large before you and seems suddenly quite menacing. Perhaps this wasn't the best idea...`
+			`	"Don't move, or we will end you!" comes a shout from within the darkness of the mine. "Identify yourself, now!"`
+			choice
+				`	"I'm Captain <last>, veteran of the human civil war!"`
+					goto timsbud
+				`	"I'm a friend of Timothy Radrickson!"`
+					goto timsbud
+				`	"I'm the Hero of Rand!"`
+					goto timsbud
+				`	"I'm just a merchant! My name's Captain <last>!"`
+					goto timsbud
+				`	"Mind your own business, I answer to no one!"`
+				`	(Run back to your ship.)`
+					goto "to the ship"
+
+			`	Your response is followed by a short moment of stunned silence, a snicker, and then the click of a trigger.`
+			`	There is a brief, almost pleasant, moment of calm as the inevitability of the situation sinks in... unfortunately, that is the last moment of calm you ever experience, as your body is riddled with bullet rounds mere seconds later.`
+				die
+
+			label timsbud
+			`	There is a brief, low conversation, followed by what sounds like a voice on a radio. Then finally, "If you're who you say you are, how did you first meet Governor Radrickson?"`
+			choice
+				`	"I ran into him at the Rand spaceport."`
+				`	"He sent me a message that he wanted to meet me."`
+				`	"I found him in an old equipment locker."`
+				`	"He was hiding in a food crate on my ship."`
+					goto right
+
+			`	You hear someone swear and say, "We gotta move now! Everything we can't carry, burn; they're onto us!" Some commotion ensues and you hear a few individuals run into the cave.`
+			`	The remaining guard hears something on the radio and then says, "No loose ends. Got it."`
+			choice
+				`	(Run after them into the cave.)`
+				`	(Run back to your ship.)`
+					goto "to the ship"
+			`	You run straight for the cave, hand on pistol and heart racing. Unfortunately for you, instead of the cave you run right into a bullet. The last thing you see is the guard's dirtied face looking over your body before he finishes the job.`
+				die
+
+			label "to the ship"
+			`	You turn around and dash back to your ship, and find that a few bullet rounds followed you as well. You quickly take off, expecting ships to come after you; however, none do.`
+				flee
+
+			label right
+			`	Despite not being able to see your potential assailants you can feel an immediate shift in the tension.`
+			`	"Good," one of them says, seemingly more to themselves than to you. "That's real good." You hear a loud sigh and then, "Alright, Hero of Rand, you're good to approach. The boss wants to see you."`
```

Comment:
This makes the invitation more implicit when First Last doesn't do the freighter mission, but still works (i.e. immediate shift in the tension -->gently ushered forward)
```suggestion
			`	"Good," one of them says, seemingly more to themselves than to you. "That's real good." You hear a loud sigh and then, "Alright, Hero of Rand, you're good to approach. The boss wants to see you."`
				to display
					has "Timothy Radrickson 3d: Return to Rand: done"
```
The later one on line 8267 I think still works regardless for Timothy being the real Hero of Rand, so I don't think a change is needed there